docs/current_docs/quickstart/853930-directories.mdx

```diff
@@ -97,13 +101,13 @@ tracing
 zenith
 ```
 
-This does look like the contents of the Dagger repository after build. Great! Next, export its contents to the local filesystem:
+This does look like the contents of the Dagger repository after build. Great! Next, export the directory to the local filesystem:
 
 ```shell
-dagger -m github.com/kpenfound/dagger-modules/golang@v0.1.5 call build --project https://github.com/dagger/dagger --args ./cmd/dagger export --path ./my-dagger-build
+dagger -m github.com/kpenfound/dagger-modules/golang@v0.1.5 call build --project=https://github.com/dagger/dagger --args=./cmd/dagger export --path=./my-dagger-build
 ```
 
-This revised command builds the Go binary and exports the build output directory to the Dagger host. Once the command completes execution, a new `my-dagger-build` directory is created in the current working directory on the Dagger host. This directory contains the compiled Go binary.
+This revised command builds the Go binary and exports the returned directory to the Dagger host. A new `my-dagger-build` directory is created in the current working directory on the Dagger host. This directory contains the contents of the Dagger repository, plus the compiled Go binary.
```

Comment:
> This directory contains the contents of the Dagger repository, plus the compiled Go binary.

I find it confusing that we're exporting the contents of the repo as well, what's the reason for that? I'm thinking if it would make more sense to modify the module itself to return only the binary?

Reply:
I think returning the directory is more flexible, since you can always chain in a `file` if you want a specific file, or nothing if you want the entire directory.

Reply:
Yeah, but you need to at least return a directory with the results of the build, e.g., a `./bin` directory, rather than the whole source dir. The golang module being used is putting the built binary in that root, so I suggest @kpenfound makes an update to fix that.

Reply:
@helderco separately from the module fix, since returning the directory seems to be a source of confusion, I adjusted the example to select and export the file only using `file...---output`.

Reply:
@helderco something like this `dagger -m github.com/kpenfound/dagger-modules/golang@v0.1.5 call build --project=https://github.com/dagger/dagger --args=-o,./bin/,./cmd/dagger directory --path bin export --path=./my-dagger-build` or are you thinking the command should be as-is but the module should return a directory with only the built artifacts?

Reply:
I think the golang module should save by default in `./bin` or some other directory that only has the built binary or binaries, and return it. That makes for simpler usage.

Reply:
I'll echo what Helder said above, that feels more intuitive to me as we use it!

Reply:
Updated: `dagger -m github.com/kpenfound/dagger-modules/golang@v0.1.10 call build --source=https://github.com/dagger/dagger --args=./cmd/dagger export
 --path=./my-dagger-build`
 
 Note that the argument `project` is renamed to `source` for the newer version

Reply:
I have updated this in the text now.